api/notd/api/gallery_v1.py

```diff
@@ -92,4 +93,9 @@ async def get_gallery_user_owned_collections(registryAddress: str, userAddress:
         ownedCollections = await galleryManager.get_gallery_user_owned_collections(registryAddress=registryAddress, userAddress=userAddress)
         return GetGalleryUserOwnedCollectionsResponse(ownedCollections=(await responseBuilder.gallery_owned_collections_from_models(ownedCollections=ownedCollections)))
 
+    @router.get('/collections/{galleryAddress}/overlaps')
```

Comment:
galleryAddress -> registryAddress

Reply:
Why registryAddress, I thought we were just running it for the gallery's

Reply:
look at all the apis in this file. we never use galleryAddress as a variable for this